aten/src/ATen/native/cuda/PowKernel.cu

```diff
@@ -84,18 +84,19 @@ void pow_scalar_tensor_impl(TensorIteratorBase& iter, c10::complex<value_t> base
 }
 
 void pow_tensor_tensor_kernel(TensorIteratorBase& iter) {
-  AT_DISPATCH_ALL_TYPES_AND_COMPLEX_AND2(
-      kHalf, kBFloat16, iter.common_dtype(), "pow_cuda", [&] {
+  AT_DISPATCH_ALL_TYPES_AND_COMPLEX_AND3(
+      kComplexHalf, kHalf, kBFloat16, iter.common_dtype(), "pow_cuda", [&] {
+    using opmath_t = at::opmath_type<scalar_t>;
```

Comment:
Have used `opmath_t` here which will update the behaviour for `float16` and `bfloat16` as well (though it should be ok I believe).

cc: @ngimel